slam-libraries/viam-cartographer/src/main_old.cc

```diff
@@ -0,0 +1,371 @@
+// This is an Experimental variation of cartographer. It has not yet been
```

Comment:
`main_old.cc` was originally the `main.cc` script. Here renamed and kept since it is still functional. Can be started using `./scripts/run_cartographer_old.sh`.